db/src/debug.rs

```diff
@@ -147,13 +155,17 @@ pub fn transactions_after<S: Borrow<Schema>>(conn: &rusqlite::Connection, schema
         let v: rusqlite::types::Value = row.get_checked(2)?;
         let value_type_tag: i32 = row.get_checked(3)?;
 
-        let typed_value = TypedValue::from_sql_value_pair(v, value_type_tag)?;
+        let typed_value = match TypedValue::from_sql_value_pair(v, value_type_tag)? {
+            TypedValue::Ref(e) => {
+                borrowed_schema.get_ident(e).cloned().map(TypedValue::Keyword).unwrap_or(TypedValue::Ref(e))
```

Comment:
You do this twice, so why not add a trait to `TypedValue` that ident-maps with a provided schema?

```rust
trait ToIdent {
  fn map_ident(self, schema: &Schema) -> Self;
}
```